test/integrationtest/projecttest/tests/graphql/testutil_test.go

```diff
@@ -148,3 +148,21 @@ func getValue(response *httpexpect.Response, queryName, key string) *httpexpect.
 func getArray(response *httpexpect.Response, queryName string) *httpexpect.Array {
 	return response.JSON().Object().Value("data").Object().Value(queryName).Array()
 }
+
+func assertResponseForExpectedState(t *testing.T, e *httpexpect.Expect, response *httpexpect.Response, expectedState map[string]interface{}) error {
```

Comment:
the assert function should directly use `t.Fail()` or similar instead of returning an error